src/main/java/se/sundsvall/casedata/integration/messageexchange/configuration/MessageExchangeConfiguration.java

```diff
@@ -24,9 +30,17 @@ JsonFormWriter jsonFormWriter() {
 	FeignBuilderCustomizer feignBuilderCustomizer(final ClientRegistrationRepository clientRepository, final MessageExchangeProperties messageExchangeProperties) {
 		return FeignMultiCustomizer.create()
 			.withErrorDecoder(new ProblemErrorDecoder(CLIENT_ID))
+			.withRequestInterceptor(builder -> builder.header(Identifier.HEADER_NAME, createSentByHeaderValue(Identifier.get())))
 			.withRequestTimeoutsInSeconds(messageExchangeProperties.connectTimeout(), messageExchangeProperties.readTimeout())
 			.withRetryableOAuth2InterceptorForClientRegistration(clientRepository.findByRegistrationId(CLIENT_ID))
 			.composeCustomizersToOne();
 	}
 
+	String createSentByHeaderValue(final Identifier identifier) {
+		return ofNullable(identifier)
+			.filter(i -> allNotNull(i.getType(), i.getValue()))
+			.map(i -> "%s; type=%s".formatted(ofNullable(i.getValue()).orElse(""), ofNullable(i.getType()).map(t -> UPPER_UNDERSCORE.to(LOWER_CAMEL, i.getType().name())).orElse("")))
```

Comment:
There is a new dept44-release on the way.
If you wait an hour, you can do this instaead:
`.map(i -> i.toHeaderValue()))`

814ba34b-5c24-4e58-beee-4586c66ab849

Reply:
Ok.

Reply:
Is it out yet?

Reply:
Soon....